dashboard/app/models/user.rb

```diff
@@ -1487,6 +1487,10 @@ def assigned_courses
     section_courses.map(&:summarize_short)
   end
 
+  def assigned?(course)
+    section_courses.include?(course)
```

Comment:
nit: could call this `user.assigned_course?(course)` to help ensure no one calls it with a script.